### PR TITLE
Remove unused deps

### PR DESCRIPTION
**Pull request by JSAssassin**
Addresses https://github.com/veres-one/did-veres-one/issues/64#issuecomment-1130224772
Removed unused deps `apisauce@2.1.1`, `jsonld@5.2.0` and devDep `@digitalbazaar/ed25519-verification-key-2018@3.1.1`

### CODE REVIEW

**Comment by dlongley** [comment]
I don't know the process for this repo. @davidlehn, can you look at this?

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```
Comment (davidlehn) [inline .npmrc:1]: Is this needed?  I see no peer deps.

**Comment by davidlehn** [comment]
I do not know what is going on with releases here.

I think @dmitrizagidulin(?) had tried to keep the main branch and default npm installs working with the live network.  The v14 branches and tags were for the next gen network.  Theory being least surprise for other developers.

I suspect this is not the best way currently.  It makes using the latest network more difficult with non-semver tags or branches.  I think we should try to keep latest development on main and add updates and releases to a v13.x (or similar) branch as needed.  We could use npm tag features to keep default installs working with live networks, but make it easier for all development work to have a normal flow with more recent versions.  I'm not sure of all the details.  Consideration must be made for how to get 14.x back to main too.

Note also the earlier unmerged dep update: https://github.com/veres-one/did-veres-one/pull/81
And the soon to happen ESM conversion that built upon it: https://github.com/veres-one/did-veres-one/pull/82

I think those have all the changes here and more?  But it also goes to ESM so users of 14.x won't see them.  I did just now have to remove `@digitalbazaar/ed25519-verification-key-2018` because apparently `npm-check` doesn't look at branch deps.

I'm not sure what the short term plan is here.  If these changes are needed for something in particular by 14.x users, we can figure out how to make a new 14.x tag.  But as can be seen in ESM branch, there's plenty more cleanup to be done.

File: .npmrc (hunk 1 above)
Comment (JSAssassin) [reply .npmrc]: Removed.

**Comment by mattcollier** [comment]
@davidlehn are you blocked on tagging/releasing this PR?

**Review by aljones15 — APPROVED** [review]

**Comment by davidlehn** [comment]
We're moving to latest releases off of `main`.  The changes here may not be needed anymore?  If they are, please rebase on main and change PR base branch to main.

**Comment by JSAssassin** [comment]
Closing.. these changes are no longer relevant since they are already on main.

**Pull request closed without merging**